src/App.tsx

```diff
@@ -1,61 +1,129 @@
+import React, { useState } from 'react';
+
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User } from './types';
+
+const getUserById = (userId: number): User | null =>
```

Comment:
Good, but you can create new folder "utils" for 2 functions: getUserById and getNewId